pkgs/tools/security/sha1collisiondetection/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, libtool }:
+
+stdenv.mkDerivation  rec {
+  pname = "sha1collisiondetection";
+  version = "git-20170221";
```

Comment:
@leenaars As far as I can see the `version = "git-20170221";` is still there. It should be `version = "2017-02-21";` also note that versions in Nix always start with a digit. So at the moment the package name, according to Nix, is "sha1collisiondetection-unstable-git" and the version is "20170221".